--- configure.orig	2013-03-22 07:27:03.000000000 -0600
+++ configure	2013-04-30 09:27:28.433956114 -0600
@@ -12015,9 +12015,6 @@
 else
 
     ax_cv_have_sse3_ext=no
-    if test "$((0x$ecx&0x01))" = 1; then
-      ax_cv_have_sse3_ext=yes
-    fi
 
 fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ax_cv_have_sse3_ext" >&5
@@ -12030,9 +12027,6 @@
 else
 
     ax_cv_have_ssse3_ext=no
-    if test "$((0x$ecx>>9&0x01))" = 1; then
-      ax_cv_have_ssse3_ext=yes
-    fi
 
 fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ax_cv_have_ssse3_ext" >&5
